src/main/resources/com/google/api/codegen/php/discovery_fragment.snip

```diff
@@ -248,8 +248,8 @@
   optParams
 @end
 
-@private requestBodyName()
-  requestBody
+@private postBodyName()
+  postBody
```

Comment:
Yeah... I think the compilecheck script is doing 1:1 comparisons of the parameter names